### PR TITLE
Introside: :memo: update confirmation text

### LINKED ISSUE #3298: Bug: Rette tekst på introside-mal
Teksten til checkbox med brukers erklæring i kodeeksemplene til infoside er feil. 
Riktig tekst er: Jeg bekrefter at jeg vil svare så riktig som jeg kan.

Dette gjelder både malen på toppen av siden og eksemplene på bunnen av siden.

### PR DESCRIPTION
**Pull request by JulianNymark**
### Description

closes https://github.com/navikt/aksel/issues/3298

### Component Checklist 📝

- [x] JSDoc
- [x] Examples
- [x] Documentation / Decision Records
- [x] Storybook
- [x] Style mappings (`@navikt/core/css/config/_mappings.js`)
- [x] Component tokens (`@navikt/core/css/tokens.json`)
- [x] CSS class deprecations (`@navikt/aksel-stylelint/src/deprecations.ts`)
- [x] Exports (`@navikt/core/react/src/index.ts` and `@navikt/core/react/package.json`)
- [x] New component? CSS import (`@navikt/core/css/index.css`)
- [x] Breaking change? Update migration guide. Consider codemod.
- [x] Changeset (Format: `<Component>: <gitmoji?> <Text>.` E.g. "Button: :sparkles: Add feature xyz.")

### CODE REVIEW

**Comment by JulianNymark** [comment]
Need to update the changelog for affected templates as well (this data is in sanity iirc?)

File: aksel.nav.no/website/pages/templates/soknad-introside-real-examples/soknad-introside-aap.tsx
Comment (HalvorHaugan) [inline aksel.nav.no/website/pages/templates/soknad-introside-real-examples/soknad-introside-aap.tsx:1]: I think you forgot to update the version number in this example.

File: aksel.nav.no/website/pages/templates/soknad-introside-real-examples/soknad-introside-pass-av-barn.tsx
Comment (HalvorHaugan) [inline aksel.nav.no/website/pages/templates/soknad-introside-real-examples/soknad-introside-pass-av-barn.tsx:1]: I think you forgot to update the version number in this example.

**Comment by KenAJoh** [comment]
> Need to update the changelog for affected templates as well (this data is in sanity iirc?)

It's defined in the `meta.json`-file located with the templates https://github.com/navikt/aksel/blob/main/aksel.nav.no/website/pages/templates/soknad-introside/meta.json

**Review by HalvorHaugan — DISMISSED** [review]
Remember to update the changelog in meta.json as well.

**Review by HalvorHaugan — COMMENTED** [review]
LGTM

**Review by KenAJoh — APPROVED** [review]

**Pull request merged**